pyproject.toml

```diff
@@ -67,3 +67,51 @@ select = [
 "**/{tests}/*" = ["B007"]
 "__init__.py" = ["F401"]
 "docs/conf.py" = ["F401"]
+
+[tool.pixi.project]
+name = "jaxsim"
+channels = ["conda-forge"]
+platforms = ["linux-64", "linux-aarch64", "osx-64", "osx-arm64"]
+
+[tool.pixi.dependencies]
+coloredlogs = "*"
+jax = "*"
+jaxlib = "*"
+jaxlie = "*"
+jax-dataclasses = "*"
+pptree = "*"
+rod = "*"
+typing_extensions = "*"
+lxml = "*"
+mediapy = "*"
+mujoco = "*"
+libgz-sim8 = "*"
+
+[tool.pixi.feature.test.tasks]
+test = "pytest"
+
+[tool.pixi.feature.test.dependencies]
+black = "24.*"
+isort = "*"
+pre-commit = "*"
+idyntree = "*"
+pytest = "*"
+pytest-icdiff = "*"
+robot_descriptions = "*"
+
+[tool.pixi.feature.gpu]
+
```

Comment:
```suggestion
[tool.pixi.feature.gpu]
```

Reply:
Nit, just blank line.

Reply:
The suggestion is not removing the blank file, probably there is a bug in GitHub's suggestion method.